src/components/common/switch-button/SwitchButton.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import Switch from '@mui/material/Switch';
+import { ThemeProvider } from '@mui/material/styles';
+import { customSwithTheme } from './switch-button-theme';
+import { SwitchButtonProps } from '../../../types/types';
+
+export default function SwitchButton({
+  checked,
+  onChange,
+}: SwitchButtonProps): React.ReactElement {
+  return (
+    <ThemeProvider theme={customSwithTheme}>
```

Comment:
It's better to wrap the whole application then you don't have to do it for each MUI element.